roles/internal/fcrepo/templates/activemq.xml.j2

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<beans xmlns="http://www.springframework.org/schema/beans"
+    xmlns:amq="http://activemq.apache.org/schema/core"
+    xmlns:context="http://www.springframework.org/schema/context"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://www.springframework.org/schema/beans http://www.springframework.org/schema/beans/spring-beans.xsd
+    http://www.springframework.org/schema/context http://www.springframework.org/schema/context/spring-context-3.0.xsd
+    http://activemq.apache.org/schema/core http://activemq.apache.org/schema/core/activemq-core.xsd">
+
+    <context:property-placeholder/>
+
+    <!--
+        The <broker> element is used to configure the ActiveMQ broker.
+    -->
+    <broker xmlns="http://activemq.apache.org/schema/core" brokerName="localhost">
+      <persistenceAdapter>
+        <kahaDB directory="${fcrepo.activemq.directory:ActiveMQ/kahadb}"/>
```

Comment:
Are we setting `fcrepo.activemq.directory` somewhere?

Reply:
I assume its being set in the fedora configs somewhere. I stole this line directly from the activemq config that ships with fcrepo. 


https://github.com/fcrepo4/fcrepo4/blob/master/fcrepo-configs/src/main/resources/config/activemq.xml#L78-L80

Reply:
Okay, cool, then it's not CLAW's problem.